### ISSUE TITLE
Which file do we run firstly?

### ISSUE THREAD

**Issue by tphankr** [lucasmansilla/ACRN_Chest_X-ray_IA#13]
Thank you for your code, @lucasmansilla.
Please guide us, which file do we run firstly? Thank you.
![image](https://user-images.githubusercontent.com/37217354/130556621-4113aa0a-adc8-4d9d-8acc-3328fe63b270.png)

Thank you.

**Comment by lucasmansilla** [comment]
Hi @tphankr,

If you want to train AC-RegNet, you first need to train the autoencoder. So, first run` train_aenet.py `and then `train_acregnet.py`.

**Comment by tphankr** [comment]
Thank you @lucasmansilla,
Allow me ask, setting the data folder to run the code, thank you @lucasmansilla.
I downloaded the NIH Chest-XRay14 dataset already. And put it in the same folder with the train_aenet.py or train_acregnet.py.
![image](https://user-images.githubusercontent.com/37217354/130715204-e9287f40-22c2-48a9-9dc6-4b09a22f48c9.png)
![image](https://user-images.githubusercontent.com/37217354/130715550-86d1b08f-ddba-4956-a7eb-b1329d66f9c9.png)
![image](https://user-images.githubusercontent.com/37217354/130715576-9df874c8-3bde-4cb0-a329-abca7a187590.png)


And then, I typed "python train_aenet.py", It is like not run, as following:
![image](https://user-images.githubusercontent.com/37217354/130715328-4d294eda-4bc0-4a7c-95b6-2e7cca9ee6e4.png)

Thank you.
